jwcrypto/jwe.py

```diff
@@ -91,7 +92,10 @@ def __init__(self, plaintext=None, protected=None, unprotected=None,
         self._allowed_algs = None
         self.objects = dict()
         self.plaintext = None
-        self.header_registry = JWSEHeaderRegistry(header_registry)
+        if header_registry:
+            self.header_registry = JWSEHeaderRegistry(header_registry)
+        else:
+            self.header_registry = JWSEHeaderRegistry(JWEHeaderRegistry)
```

Comment:
Why this change ?

Reply:
`pep8` complained about a dictionary as a default value... Since it's mutable, and unfortunately, there isn't a frozen version of dictionary that it recognizes afaik.